src/components/GoogleAddressLookup/styles.scss

```diff
@@ -0,0 +1,36 @@
+/* stylelint-disable no-descending-specificity */
+@import '../../styles/margins.scss';
+@import '../../styles/paddings.scss';
+
+.rainbow-google-address-lookup_container {
+    .rainbow-lookup_input-icon {
+        svg {
+            width: 24px !important;
+            height: 24px !important;
+            font-size: 24px !important;
+        }
+    }
+
+    .rainbow-google-address-poweredby_container {
+        display: -ms-flexbox;
+        display: flex;
+        -ms-flex-pack: end;
+        justify-content: end;
+        -ms-flex-align: end;
+        align-items: flex-end;
+        padding-top: $padding-xx-small;
+        padding-right: $padding-x-small;
+
+        img {
+            height: 0.625rem;
+        }
+    }
+
+    .rainbow-google-address-lookup_search-option-label {
+        color: #a4a7b5;
+
+        b {
```

Comment:
We should not apply the style to the tag names, we should use classes instead. Let's follow the principle of keeping things decoupled to ensure that our components are scalable and thus prevent any change in the future from breaking them.